model/transformers/examples/qa/accuracy.sh

```diff
@@ -0,0 +1,11 @@
+dffml accuracy \
+  -model qa_model \
+  -model-model_type bert \
+  -model-save_steps 3 \
+  -model-model_name_or_path bert-base-cased \
+  -model-output_dir qamodel/checkpoints \
+  -model-cache_dir qamodel/cache \
+  -sources s=op \
+  -source-opimp model.transformers.dffml_model_transformers.qa.utils:parser \
```

Comment:
We should be able to load it with this syntax, the syntax you've used assumes the user is running from the root of the DFFML source tree.

```suggestion
  -source-opimp dffml_model_transformers.qa.utils:parser \
```